module/support/logging/FileLogHandler/data/config/docker/FileLogHandler.yaml

```diff
@@ -0,0 +1 @@
+log_file: /home/nubots/NUbots/log
```

Comment:
Why is this a part of this PR?

Reply:
Probably isn't very necessary, unless I also add a keyboardwalkfake role